### ISSUE TITLE
V3 specs not independent from each other (Deployment and cancelling deployment) 

### ISSUE THREAD

**Issue by renelehmann** [cloudfoundry/cf-acceptance-tests#531]
## Issue
Tests for "deployments" and "v3" features includes creating and removing of a zip file (staticfile.zip).
If the zip file already/still exists an error on creating the zip file will occur:
"file already exists: assets/staticfile.zip"

## Context
cats version: 7.4.0  (against cloudfoundry v19.0.0)
cats.json inlcudes v3 and deployments
running testsuite in parallel (-nodes=x) and without randomize.

The spec "Deployment" and "cancelling deployments" does/could interfere each other on having the same zip file name
which will be created on the "BeforeEach" and removed on the "AfterEach" step.
https://github.com/cloudfoundry/cf-acceptance-tests/blob/v7.4.0/v3/deployment.go.

error on (by creating the zip file if its already/still exists):
https://github.com/cloudfoundry/cf-acceptance-tests/blob/a6e4d7ae3b3360b6dbe0550ab1ee55a4771d38de/v3/deployment.go#L231

## Possible Fix
Creating dedicated/independent zip filenames for each spec or else.

workaround:
- Running cats testsuite with "randomizeAllSpecs" will probably help to prevent the situation on interfere of each other but without a guarantee.
- Running cats testsuite without parallelize (default -node=1).

**Comment by jochenehret** [comment]
Deployment tests have been refactored: https://github.com/cloudfoundry/cf-acceptance-tests/pull/1268
This issue is no longer relevant, closing.
